### DISCUSSION TITLE
Upscayl Decording error problem

### DISCUSSION THREAD

**Discussion by techmohin** [upscayl/upscayl#684] [Q&A]
Hello sir, Recently i downloaded and installed upscayl on my laptop. when I try to upscale any image it shows a decoding error problem. please help me to solve this issue. i also attach a screenshot about this exact problem.
my laptop is configured:Core i 7 3genram: 4Hd: 500gbSSD: 120 GbWindows 10

please help me to slove me this problem as soon as possible
![Screenshot_49](https://github.com/upscayl/upscayl/assets/158029191/5ddeec21-4f6b-49e6-9a48-968b8b059d85)

**Comment by NayamAmarshe** [comment]
Please follow the troubleshooting guide: https://github.com/upscayl/upscayl/wiki/Troubleshooting

**Comment by aaronliu0130** [comment]
Specifically, the FAQ.
